src/SFA.DAS.Funding.SystemAcceptanceTests/Helpers/WaitHelper.cs

```diff
@@ -8,7 +8,7 @@ public static async Task WaitForIt(Func<bool> lookForIt, string failText)
         {
             if (lookForIt()) return;
 
-            await Task.Delay(TimeSpan.FromMilliseconds(10));
+            await Task.Delay(TimeSpan.FromMilliseconds(100));
```

Comment:
Agree 10 is way too low, what made you land on 100 out of interest?

Reply:
This is could be tweaked up or down. Using 100 milliseconds didn't slow the test execution but does rest the cpu more. We could experiment further or how much slow we make it before it impacts test times. Also, without calling the multimedia api (winMM.dll) the closest resolution the system could manage is 15.625